validator/validations/steps_test.go

```diff
@@ -108,7 +175,16 @@ func TestSteps_String(t *testing.T) {
 		s    Steps
 		want string
 	}{
-		// TODO: Add test cases.
+		{
+			name: "test",
+			s:    255,
+			want: "11111111",
```

Comment:
In this case it's fine to keep the test as is, otherwise you make it more fragile. So don't change this to flags.